src/templates/template.h

```diff
@@ -308,6 +308,11 @@ Q_OBJECT
 	 */
 	void unloadTemplateFile();
 	
+	/**
+	 * Release georeferencing flag for the template file.
+	 * Must not be called before the original georeferencing information is available.
+	 */
+	void releaseGeoreference();
```

Comment:
I would prefer this to be place in the vicinity of `isTemplateGeoreferenced()`/`isTemplateGeoreferenced`.